graph/build.rs

```diff
@@ -18,4 +18,11 @@ fn main() {
         .out_dir("src/substreams")
         .compile(&["proto/substreams.proto"], &["proto"])
         .expect("Failed to compile Substreams proto(s)");
+
+    tonic_build::configure()
+        .protoc_arg("--experimental_allow_proto3_optional")
+        .extern_path(".sf.substreams.v1", "crate::substreams")
```

Comment:
no, the sf.substreams.v1 still exists for the packages, manifests and such. Only, the service layer in it has been deprecated and moved to sf.substreams.rpc.v2 (the v2 to avoid confusion, even if we change namespace)
this extern_path directive informs the rpcv2 tonic handler about where to find the sf.substreams.v1 stuff.